spot_driver/test/CMakeLists.txt

```diff
@@ -130,6 +130,8 @@ target_include_directories(test_kinematic_service
 )
 target_link_libraries(test_kinematic_service spot_api)
 
+ament_add_pytest_test(spot_driver_pytest ${CMAKE_CURRENT_SOURCE_DIR} TIMEOUT 900)
```

Comment:
Do we want to go with this approach of 900s for all of the tests or the one below which loops over and gives individual tests targets?

Reply:
To be clear I am not leaning one way or the other. Just checking

Reply:
i think just this one since it seems to be the only (relatively) consistent flaky test

Reply:
I mean do we want one target for all python tests or one target for each python test. The former has us set a timeout for all tests. The later lets use define a timeout for each test file.

Reply:
ohhhh, ok i misunderstood 
I think one target for all tests is fine